packages/features/ee/workflows/lib/aphanumericSenderIdSupport.ts

```diff
@@ -0,0 +1,75 @@
+export const noAlphanumericSenderIdSupport = [
```

Comment:
This file also is missing the 'l' in 'alpha'

Reply:
Sorry for the spelling mistakes. Thank you for pointing them out 🙏